terraform/aws/control-plane/modules/ecs/variables.tf

```diff
@@ -38,3 +38,8 @@ variable "command" {
   type        = list(string)
   default     = []
 }
+
+variable "cloudWatch_logs" {
+  description = "Control Plane CloudWatch Logs."
+  type        = bool
+}
```

Comment:
should we put the default to `true`?
As this should be the default for inexperienced users? (to be able to help them)

Reply:
The downside, and the reason I didn't set it to `true` by default, is that the log group created will continue to grow in size over time, and by default, the retention period is set to "Never expire." It's easy for those who want logs to uncomment the option. WDYT?

Reply:
Never expire is 6 months for AWS CloudWatch... so...

Reply:
I didn't know that that Never expire means 6 months on CloudWatch. This is what I found on Amazon CloudWatch:

> Log retention – By default, logs are kept indefinitely and never expire. You can adjust the retention policy for each log group, keeping the indefinite retention, or choosing a retention period between 10 years and one day.
https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/WhatIsCloudWatchLogs.html#cloudwatch-logs-features

Either way the more experienced end user can easily uncomment the activation flag and set it to false.

Reply:
Ok, maybe I assumed 6 months was the default default :)

I heard Davy telling that our logs are only kept 6 months. my bad